Index: main.py
===================================================================
diff --git a/main.py b/main.py
--- a/main.py	
+++ b/main.py	
@@ -3,7 +3,6 @@
 
 from player import player
 from settings import Settings
-import time
 
 settings = Settings()
 
@@ -21,46 +20,16 @@
         json.dump(data, g_i)
 
 
-def is_training() -> bool:
-    while True:
-        print('Пройти обучение: Y', 'Не проходить обучение: N', sep='\n')
-        string = input().lower()
-        if string == 'y':
-            return True
-        elif string == 'n':
-            return False
-
-
-def training(star_ship):
-    print(f'Добро пожаловать, {player.name}!\n'
-          f'В начале игры вам доступен {star_ship.name}.')
-    time.sleep(2)
-    print('Давай заправимся.')
-    while not star_ship.refuel(int(input('Введите кол-во топлива, которое хотите заправить: '))):
-        continue
-    print('Заправляемся...')
-    time.sleep(3)
-    print('Теперь - купим продукты, чтобы продать их на другой планете.')
-    time.sleep(2)
-    print(f'У вас {player.money} денег.')
-    time.sleep(2)
-    print('Список продуктов:')
-    star_ship.location.stock.product_list()
-    while not star_ship.buy(input('Введите название продукта, который хотели бы купить: '),
-                            int(input('И его кол-во: '))):
-        continue
-
-
 def interface(status: dict, star_ship):
     if status['new_game']:
-        if is_training():
-            training(star_ship)
-    change_file('new_game', False)
-    print()
-    print('1. Заправиться.', '2. Купить продукты.', '3. Продать продукты. ', '4. Отправиться на другую планету.',
-          '5. Посмотреть статистику игры.', '6. Выход из игры.',
+        change_file('new_game', False)
+    print('\n1. Заправиться.', '2. Купить продукты.', '3. Продать продукты. ', '4. Отправиться на другую планету.',
+          '5. Посмотреть статистику игры.', '6. Выход из игры.' + '\n',
           sep='\n')
-    print()
+    input_processing(status, star_ship)
+
+
+def input_processing(status, star_ship):
     req = int(input())
     if req == 1:
         while not star_ship.refuel(int(input('Введите кол-во топлива, которое хотите заправить: '))):
@@ -77,8 +46,7 @@
             continue
     elif req == 4:
         print('1. Посмотреть цены на продукты на других планетах.', '2. Посмотреть расстояние до планет.',
-              '3. Выбор планеты, на которую вы полетите.', '4. Вернуться на прошлую страницу.', sep='\n')
-        print()
+              '3. Выбор планеты, на которую вы полетите.', '4. Вернуться на прошлую страницу.\n', sep='\n')
         req1 = int(input())
         if req1 == 1:
             for planet in settings.planets:
@@ -91,15 +59,12 @@
             for planet in settings.planets:
                 if planet != star_ship.location:
                     print(
-                        f'От планеты {star_ship.location.name} до планеты {planet.name} {star_ship.get_distance(planet)}')
-            print()
+                        f'От планеты {star_ship.location.name} до планеты {planet.name} {star_ship.get_distance(planet)}\n')
         elif req1 == 3:
             for planet in settings.planets:
                 if planet != star_ship.location:
-                    print(planet.name)
-            print()
+                    print(planet.name + '\n')
             planet = input('Введите название планеты, на которую хотите полететь.\n')
-            print()
             for i in settings.planets:
                 if planet.lower() == i.name.lower():
                     star_ship.move_to_planet(i)
@@ -107,7 +72,10 @@
         elif req1 == 4:
             interface(status, star_ship)
     elif req == 5:
-        print(f'Вы находитесь на планете {star_ship.location.name}\n\nУ вас {player.money} кредитов\n\n{star_ship.number_of_products()}\n{star_ship.capacity - star_ship.current_capacity} свободного места в грузовом отсеке.\n\n{star_ship.tank.fuel} топлива.\n')
+        print('Список ваших продуктов.')
+        star_ship.number_of_products()
+        print(
+            f'\nВы находитесь на планете {star_ship.location.name}\nУ вас {player.money} кредитов\n{star_ship.capacity - star_ship.current_capacity} свободного места в грузовом отсеке.\n{star_ship.tank.fuel} топлива.')
     elif req == 6:
         sys.exit()
 
@@ -125,7 +93,23 @@
     return star_ship
 
 
-def is_new_game() -> tuple or bool:
+def continue_game():
+    data = read_file()
+    for i in range(len(settings.star_ships)):
+        if settings.star_ships[i].name == data['star_ship_name']:
+            star_ship = settings.star_ships[i]
+
+    for i in range(len(settings.planets)):
+        settings.planets[i].coord = data['planets_coord'][i]
+        if settings.planets[i].name == data['location']:
+            star_ship.location = settings.planets[i]
+
+    star_ship.tank.fuel = data['star_ship_fuel']
+    star_ship.cargo = data['cargo']
+    return star_ship
+
+
+def is_new_game() -> tuple:
     while True:
         print('Новая игра: Y', 'Продолжить игру: C', 'Завершить игру: N', sep='\n')
         string = input().lower()
@@ -134,18 +118,7 @@
             star_ship = new_game()
             return True, star_ship
         elif string == 'c':
-            star_ship = None
-            change_file('new_game', False)
-            for i in range(len(settings.star_ships)):
-                if settings.star_ships[i].name == read_file()['star_ship_name']:
-                    star_ship = settings.star_ships[i]
-            for i in range(len(settings.planets)):
-                settings.planets[i].coord = read_file()['planets_coord'][i]
-            for i in range(len(settings.planets)):
-                if settings.planets[i].name == read_file()['location']:
-                    star_ship.location = settings.planets[i]
-            star_ship.tank.fuel = read_file()['star_ship_fuel']
-            star_ship.cargo = read_file()['cargo']
+            star_ship = continue_game()
             return True, star_ship
         elif string == 'n':
             return False, None
@@ -157,4 +130,5 @@
         interface(read_file(), star_ship)
 
 
-main()
+if __name__ == '__main__':
+    main()
Index: game_info.json
===================================================================
diff --git a/game_info.json b/game_info.json
--- a/game_info.json	
+++ b/game_info.json	
@@ -1,1 +1,1 @@
-{"planets_coord": [[3, 3], [2, 3], [3, 1]], "new_game": false, "star_ship_name": "Buran", "star_ship_fuel": 100, "location": "Earth", "cargo": {"minerals": 0, "medicines": 0, "food": 0, "materials": 0, "appliances": 0, "technic": 0, "luxuries": 10}, "money": 8000, "nick": null}
\ No newline at end of file
+{"planets_coord": [[4, 1], [2, 5], [7, 0]], "new_game": false, "star_ship_name": "Buran", "star_ship_fuel": 100, "location": "Earth", "cargo": {"minerals": 0, "medicines": 0, "food": 0, "materials": 0, "appliances": 0, "technic": 0, "luxuries": 0}, "money": 9000, "nick": null}
\ No newline at end of file
